docs/src/pages/getting-started/installation/dependencies.react.mdx

```diff
@@ -21,13 +21,33 @@ yarn add aws-amplify @aws-amplify/ui-react
 
 Wrap your application in `<AmplifyProvider/>` and include default styling:
 
+<Tabs>
+<TabItem title="React" >
 ```jsx
 import { AmplifyProvider } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css'; // default theme
 
-export default () => (
-  <AmplifyProvider>
-    <App />
-  </AmplifyProvider>
-);
+export default function App() {
+  return (
+    <AmplifyProvider>
+      <App />
+    </AmplifyProvider>
+  );
+}
```

Comment:
Changed arrow function to named function here, because arrow/anonymous functions breaks `react/display-name` eslint [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/display-name.md#rule-details).

This was turned on by default on Next.js, and it helps debugging in general by naming your component.

Reply:
This is a good practice for us to follow when building primitives versus adding `xxx.displayName=` manually. The display name can be inferred from the name of the function <- https://reactjs.org/docs/react-component.html#displayname